gatsby-node.js

```diff
@@ -92,4 +92,10 @@ exports.onCreatePage = ({ page, actions }) => {
     page.matchPath = "/sign_up/*";
     actions.createPage(page);
   }
+
+  // cart recovery
+  if (page.path.match(/^\/checkout/)) {
+    page.matchPath = "/checkout/*";
+    actions.createPage(page);
+  }
```

Comment:
All good here!

Just want to confirm that the thank you page and any required GTag events are also good to go